### ISSUE TITLE
Step 2 of 3: VB -  Tissue  Arterial  - Initialise with step 1 Using fabber - Taking too much time

### ISSUE THREAD

**Issue by ampaschoal** [ibme-qubic/oxasl#14]
Hi all,

I was using the oxasl with FSL version 6.0.1 and it was running well. Last week I updated my FSL to 6.0.4 and when using oxasl (or oxford_asl), the Step 2 of BASIL analysis is taking too much time, so much that I cant finish (one hour for 1%). Here is the print of my try. I am using Ubuntu 18.04.

![Screenshot from 2020-09-11 15-35-41](https://user-images.githubusercontent.com/38541330/92961145-280bbc80-f445-11ea-8992-b0d443c64b6e.png)

**Comment by ampaschoal** [comment]
After installing back the 6.0.1 version of FSL, I could run again the oxasl.

**Comment by mcraig-ibme** [comment]
Sorry for the slow response, I have not been getting Github notifications on this repository. I am not aware of any problems with Fabber in FSL 6.0.4. Was the first step (which seems to have completed) also very slow, or did that finish in reasonable time?

**Comment by ampaschoal** [comment]
No, the first step was quick. The second step I couldn't finish. I waited for a few hours and it still was at the beginning. I have no clue why. But after installing back an older version of FSL it worked, so I didn't try again on version 6.0.4.

**Comment by mcraig-ibme** [comment]
My best guess is that there was a numerical instability in the model. This can cause Fabber to fill up its logfile with huge amounts of data which would certainly slow it down a lot. If you have a data sample that you are able to share then I might be able to investigate and reproduce it. The issue definitely looks to be in fabber itself which would fit with the change in behaviour on installing a different version of FSL.

**Comment by robbwh** [comment]
I similarly can only get the arterial inference to work (within a reasonable timeframe) when using FSL 6.0.1.

Using FSL 6.0.5.1 it runs so slowly that I doubt it would complete within 24 hours.

The reason for this is unclear to me. Using the [fmrib example multi-pld data](https://users.fmrib.ox.ac.uk/~chappell/asl_primer/ex3/index.html), oxasl/oxford_asl run will complete just fine regardless of version. 

I am unable to share data currently, but the data come from a test subject acquired using the ADNI4 multi-pld pCASL scan card.
